includes/ApplHandlingByStr.php

```diff
@@ -33,6 +34,46 @@ function showStatus( $stat_str )
 		{
 			echo "<td><b>REJECTED</b> by ".$str[0] ."</td>";
 		}
+		else if( $str[1] == 'F' )
+		{
+			echo "<td><b>FORWARDED</b> from ".$str[0]."</td>";
+		}
+		else
```

Comment:
__[code_sniffer]__
* Spaces must be used to indent lines; tabs are not allowed
* Expected 1 space after ELSE keyword; newline found

<!-- Comment by [SideCI](https://sideci.com) -->